markdown-docs/asf_search/ASFProduct.md

```diff
@@ -84,6 +86,18 @@ None
 
 **returns:**
 
-- `shapely.geometry.point.Point` object describing the centroid of the product
+- ```shapely.geometry.point.Point``` object describing the centroid of the product
 
 <!-- Will have more than geojson export; add this when other output options available -->
+
+### <span style="color: #236192; font-size: 20px;">remotezip()</span>
+
+Returns a configured RemoteZip object, which allows downloading selected parts of a product's zip archive.
+For more information on how to use remotezip with asf-search, see [the an example in the asf-search's jupyter notebook](https://github.com/asfadmin/Discovery-asf_search/blob/master/examples/5-Download.ipynb). For more information on the open-source remotezip package, https://github.com/gtsystem/python-remotezip.
```

Comment:
Can you turn the github URL into a link? And fix the typo on the Jupyter notebook "the an example"